dashboard/src/containers/RoutesContainer/Routes.tsx

```diff
@@ -25,19 +25,19 @@ import ServiceInstanceViewContainer from "../../containers/ServiceInstanceViewCo
 type IRouteComponentPropsAndRouteProps = RouteProps & RouteComponentProps<any>;
 
 const privateRoutes = {
-  "/apps/ns/:namespace": AppListContainer,
-  "/apps/ns/:namespace/:releaseName": AppViewContainer,
-  "/apps/ns/:namespace/new/:repo/:id/versions/:version": AppNewContainer,
-  "/apps/ns/:namespace/upgrade/:releaseName": AppUpgradeContainer,
-  "/catalog/ns/:namespace": CatalogContainer,
-  "/catalog/ns/:namespace/:repo": CatalogContainer,
+  "/ns/:namespace/apps": AppListContainer,
+  "/ns/:namespace/apps/:releaseName": AppViewContainer,
+  "/ns/:namespace/apps/:releaseName/upgrade": AppUpgradeContainer,
+  "/ns/:namespace/apps/new/:repo/:id/versions/:version": AppNewContainer,
+  "/ns/:namespace/catalog": CatalogContainer,
+  "/ns/:namespace/catalog/:repo": CatalogContainer,
```

Comment:
I guess we would need to change also the routes for operators (line 70). I can take care of that